accounts-db/src/accounts_db/tests.rs

```diff
@@ -25,6 +25,8 @@ use {
     test_case::{test_case, test_matrix},
 };
 
+const DEFAULT_FILE_SIZE: u64 = 4 * 1024 * 1024;
```

Comment:
I moved this constant over from `accounts_db.rs` since some tests reference it. We can remove it later if we want, I wanted to keep the PR's diff minimal.